src/App.js

```diff
@@ -80,7 +85,18 @@ class Root extends React.Component {
   render() {
     if (this.props.isVersionAllowed === undefined) {
       // Waiting for version
-      return null;
+      return (
+        <Router>
+          <>
```

Comment:
Why do you need this? `<>`

Reply:
This is a syntax sugar for `React.fragment`. So, we can have just one child of `Router` without get html dirty with a meaningless `div`. 
And we need an element there because: `A <Router> may have only one child element`.

Ref: https://reactjs.org/docs/fragments.html